client/src/util/dataframe/dataframe.ts

```diff
@@ -242,14 +265,12 @@ class Dataframe {
     };
 
     /* test for row label inclusion in column */
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-    const has = function has(rlabel: any) {
-      const offset = getRowByOffset(rlabel);
-      return offset >= 0 && offset < length;
+    const has = function has(rlabel: LabelType) {
+      const offset = getRowOffset(rlabel);
+      return offset !== undefined && offset >= 0 && offset < length;
```

Comment:
super nit: do we need to check `offset !== undefined` if `getRowOffset()` always returns a number?

Reply:
good catch - that is a BUG!!!

Reply:
Hurray! Do we need code change here? It seems to still be `offset !== undefined && offset >= 0 && offset < ` 😆

Reply:
Only because the comment refers to the outdated code.    Take a look at the most recent commit to the PR, not the obsolete commit :-)